.circleci/config.yml

```diff
@@ -430,6 +431,60 @@ jobs:
       - store_artifacts:
           path: /home/circleci/project/test-reports/report.htm
 
+  integration_test_ios_fennec:
+    macos:
+      xcode: 15.3.0
+    steps:
+      - checkout
+      - macos/preboot-simulator:
+          version: "17.4"
+          platform: "iOS"
+          device: "iPhone 15"
+      - attach_workspace:
+          at: /tmp/experimenter
```

Comment:
Will this work for forks?

Reply:
Confirmed working on forks, [link](https://app.circleci.com/pipelines/github/jrbenny35/experimenter/826/workflows/193a7534-b6eb-4f76-8f5d-ec1c1bed90ff/jobs/1127)